packages/plugins/RSS3/src/SNSAdaptor/hooks/useFootprints.ts

```diff
@@ -1,8 +1,7 @@
-import { RSS3, RSS3BaseAPI } from '@masknet/web3-providers'
+import { RSS3 } from '@masknet/web3-providers'
 import { useAsync } from 'react-use'
-import type { AsyncState } from 'react-use/lib/useAsync'
 
-export function useFootprints(address: string): AsyncState<RSS3BaseAPI.Collection[] | undefined> {
+export function useFootprints(address: string) {
```

Comment:
Does it necessary to query with a `chainId`?

Reply:
ditto.